API.md

```diff
@@ -971,6 +971,54 @@ If there was an error, expect either an HTTP status code in the 4XX or 5XX range
 }
 ```
 
+### modify_replica
+
+Modifies properties of a single replica.
+
+#### Request
+
+```bash
+curl http://localhost:<port>/irods-http-api/<version>/data-objects \
+    -H 'Authorization: Bearer <token>' \
+    --data-urlencode 'op=modify_replica' \
+    --data-urlencode 'lpath=<string>' \ # Absolute logical path to a data object.
+    --data-urlencode 'resource-hierarchy=<string>' \
+    --data-urlencode 'replica-number=<integer>' \
+    --data-urlencode 'new-data-checksum=<string>' \
+    --data-urlencode 'new-data-comments=<string>' \
+    --data-urlencode 'new-data-create-time=<integer>' \
+    --data-urlencode 'new-data-expiry=<integer>' \
+    --data-urlencode 'new-data-mode=<string>' \
+    --data-urlencode 'new-data-modify-time=<string>' \
+    --data-urlencode 'new-data-path=<string>' \
+    --data-urlencode 'new-data-replica-number=<integer>' \
+    --data-urlencode 'new-data-replica-status=<integer>' \
+    --data-urlencode 'new-data-resource-id=<integer>' \
+    --data-urlencode 'new-data-size=<integer>' \
+    --data-urlencode 'new-data-status=<string>' \
+    --data-urlencode 'new-data-type-name=<string>' \
+    --data-urlencode 'new-data-version=<string>'
```

Comment:
Any thoughts on this?

Reply:
I not crazy about the `new-` prefix either, but for the sake of time, it felt good enough to make it clear to users what the params mean.

Reply:
Updated documentation.